site/content/docs/CICD/vercel.md

```diff
@@ -125,49 +125,58 @@ This is how a Checkly check maps to a Vercel check:
 
-A Browser check — when using the Playwright library — automatically [collects and reports Web Vital metrics](/docs/browser-checks/tracing-web-vitals/#performance-tracing-with-web-vitals). 
-These Web Vitals are reported to Vercel as the separate Performance check.
+![blocking deployments](/docs/images/cicd/vercel/vercel_ves.png)
 
-Together with the team at Vercel, we developed some custom logic to **block** deployments if any Web Vitals on your project
-degrade and cross from a "superior" range to a "inferior". For example, this could be going from **good** to **needs improvement**. 
-These ranges are listed in the table below and are based on the guidelines developed by the [Web Vitals maintainers at Google.](https://web.dev/learn-web-vitals/)
+The Virtual Experience Score does three things:
 
-| KPI | good       | needs improvement  | poor     |
-|-----|------------|--------------------|----------|
-| FCP | <= .93 sec | > .93 sec <= 1.6 sec | > 1.6 sec  |
-| LCP | <= 1.2 sec | > 1.2 sec <= 2.4 sec | > 2.4 sec  |
-| CLS | <= 0.1     | > 0.1  <= 0.25     | > 0.25   |
-| TBT | <= 150 ms  | > 150 ms <= 350 ms | > 350 ms |
+1. It aggregates your Web Vitals metrics into one number from 0 to 100, just like you might have seen with [Lighthouse scoring](https://web.dev/performance-scoring/).
+2. It assigns a "weight" to each score, marking its impact on your UX.
+3. It chops up the score into the top 10% (p10), the top 50% (median) and the rest.
 
+In the table below you can see the exact numbers their weight in the Virtual Experience Score.
 
-We will report a Browser Performance check as **passing** when:
+| Web Vitals metrics | p10 / 90%    | median / 50% | weight |
+|--------------------|--------|--------|--------|
+| FCP                | 934ms  | 1600ms | 0.20   |
+| LCP                | 1200ms | 2400ms | 0.35   |
+| TBT                | 150ms  | 350ms  | 0.30   |
+| CLS                | 0.1    | 0.25   | 0.15   |
 
-1. The Web Vitals measurement on a previous deploy is missing, but the current deployment does have measurements; we assume the new score is better.
-2. The Web Vitals measurements are within the target range.
 
-We will report it as **blocking** when:
+So, what does this mean for the blocking heuristics? It means that we will mark a check as **blocking** when:
 
 1. A check reports web vitals, and...
-2. The web vitals have degraded in relation to the last available **Production** deploy. Degraded means they passed the range threshold, i.e from **poor** to **needs improvement**.
+2. The VES has degraded in relation to the last available **Production** deploy. Degraded means they passed the range threshold, i.e from **p10** to **median**. 
+
+For more info on the **Virtual Experience Score** [check the documentation on the Vercel site](https://vercel.com/docs/concepts/analytics/web-vitals#virtual-experience-score)
```

Comment:
Period at end of sentence? I feel you like to skip these after links as they don't look as good somehow.